### DISCUSSION TITLE
File uploading has broken down

### DISCUSSION THREAD

**Discussion by birdie-github** [community/community#113429] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Issues

### Body

Hi,

I've now uploaded three images to a GitHub ticket and all three fail to render/show up.

Here's the last one:

https://user-images.githubusercontent.com/4958603/314475409-7e6c8091-c472-484f-8ab1-25b6f8a94b8b.gif

```XML
<?xml version="1.0" encoding="UTF-8"?>
<Error>
	<Code>AccessDenied</Code>
	<Message>Access Denied</Message>
	<RequestId>TNEMMN5HEQASKW4D</RequestId>
	<HostId>ZPwWEsqS9xYE1i7GqfdvcgZ+hl12DZvIwJ61eeaTa0VjnYLxEwxh02ccrSY0xR3mLZA1aiL5J4g=</HostId>
</Error>
```
It must be embedded in https://gist.github.com/probonopd/9feb7c20257af5dd915e3a9f2d1f2277?permalink_comment_id=4994812#gistcomment-4994812 but it fails to load.

Please fix ASAP.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @birdie-github ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
